user-stories/frontend/US-106-file-retrieval-interface.md

```
@@ -0,0 +1,117 @@
+    I -->|Yes| K{Authorized?}
+    K -->|No| L[Show 403 Error]
+    K -->|Yes| M[Show File Info]
+    M --> N{Previewable?}
+    N -->|Yes| O[Show Preview]
+    N -->|No| P[Show Download Only]
+    O --> Q[Download Option]
+    P --> Q
+```
+
+## Wireframe
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)